### ISSUE TITLE
Yarn 2 warning

### ISSUE THREAD

**Issue by hacknug** [gridsome/eslint-plugin-gridsome#57]
I was just trying out Yarn 2 to see if there were any major issues with Gridsome and got this warning:
```zsh
eslint-plugin-gridsome@npm:1.4.0 doesn't provide eslint@^5.0.0 || ^6.0.0 requested by vue-eslint-parser@npm:6.0.5
```

Guessing `eslint` should be listed either in `dependencies ` or `peerDependencies `? 🤔

**Comment by tyankatsu0105** [comment]
@hacknug
Umm...
Maybe we need to add `engines` to `package.json`.
Like this:

```json
{
  "engines": {
    "yarn": "^1.22.0"
  }
}
```

> 1.22.0 is current's stable version of Yarn.

**Comment by hacknug** [comment]
I don't think that's the way to solve it since people should (ideally) be able to also use `npm` and `pnpm` to manage their dependencies.

> Guessing `eslint` should be listed either in `dependencies` or `peerDependencies`? 🤔

I think it's related to how deps are listed, like I mentioned. Maybe @hjvedvik can help us out here since he probably has more experience dealing with this things than I do haha
